x-pack/plugins/apm/server/routes/service_groups/route.ts

```diff
@@ -39,6 +38,67 @@ const serviceGroupsRoute = createApmServerRoute({
   },
 });
 
+const serviceGroupsWithServiceCountRoute = createApmServerRoute({
+  endpoint: 'GET /internal/apm/service-groups-services-count',
```

Comment:
I think we use snake_case on route endpoints, don't we?

```suggestion
  endpoint: 'GET /internal/apm/service_groups_services_count',
```

Reply:
We are not really consistent unfortunately...